.github/workflows/backup.yml

```diff
@@ -0,0 +1,48 @@
+---
+name: Create external backup
+
+# yamllint disable-line rule:truthy
+on:
+  schedule:
+    # Run every day at 10am
+    - cron: '0 10 * * *'
+  workflow_dispatch:
+
+jobs:
+  create-backup:
+    runs-on: ubuntu-22.04
+    defaults:
+      run:
+        shell: bash
+        working-directory: backup
+    env:
+      RCLONE_HOST: ${{ secrets.RCLONE_HOST }}
+      RCLONE_USER: ${{ secrets.RCLONE_USER }}
+      RCLONE_PASS: ${{ secrets.RCLONE_PASS }}
+      BORG_SSHKEY: ${{ secrets.BORG_SSHKEY }}
+      BORG_REPO: ${{ secrets.BORG_REPO }}
+      BORG_PASS: ${{ secrets.BORG_PASS }}
+      FTP_DIR: ostra
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install prerequisites and create initial configuration
+        run: |
+          sudo apt-get update
+          sudo apt-get install borgbackup
+          sudo pip install borgmatic
+          curl -O https://downloads.rclone.org/rclone-current-linux-amd64.zip
```

Comment:
Tohle jediné mi přijde křehké.

Neověřujeme otisk a po čase čekám, že se kombinace `ubuntu-22.04` a `current-linux` rozbije.

Na mém Linux Mint 21, což je z velké části Ubuntu 22.04 a dost repositářů přebírá, mi to jde nainstalovat jako běžný balíček.
V Dockeru s Debian 11 taky.
Jde o to mít nejposlednější verzi?
V Debianu mi to nainstalovalo `v1.53.3-DEV` a to nezná parametr `--non-interactive`

Můžeme si dovolit to instalovat standardně?

```
sudo apt-get install rclone
```

Reply:
Já bych se toho zas tak nebál, až se to rozbije tak to buď opravíme, nebo to nahradíme odkazem na poslední funkční verzi. Do té doby ať si to klidně stahuje nejnovější stable, to Ubuntu 22.04 tam snad taky nebude navždy. Z distribučního repositáře jsem to samozřejmě zkoušel jako první, ale je 1.53 je bohužel příliš obstarožní, je potřeba alespoň 1.55 (kvůli šifrovanému spojení na FTP).

Reply:
> Z distribučního repositáře jsem to samozřejmě zkoušel jako první, ale je 1.53 je bohužel příliš obstarožní, je potřeba alespoň 1.55 (kvůli šifrovanému spojení na FTP).

Tohle je pádný důvod.